components/support/migration/metrics.yaml

```diff
@@ -0,0 +1,508 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+migration:
+  migration_versions:
```

Comment:
nit: `migration` is redundant in this case, since the category is already called `migration`. The final metric name in the ping would be `migration.migration_version`, which seems.. awkward.

Reply:
I did this to make sure "versions of migrations" aren't understood as "migration of versions". Both are a little awkward in their own ways :)

Reply:
I see, ok!